core-utilities/src/main/java/edu/illinois/cs/cogcomp/annotation/XmlTextAnnotationMaker.java

```diff
@@ -71,7 +71,6 @@ public XmlTextAnnotationMaker(TextAnnotationBuilder taBuilder, XmlDocumentProces
      */
     public XmlTextAnnotation createTextAnnotation(String xmlText, String corpusId, String docId)  {
 
-        logger.info("processing text from document {}", docId);
```

Comment:
I consider this white noise. Maybe at trace level, but when reading bunches of files, you often fill your log with these messages.